src/Lykke.Service.Qtum.Api.Core/Helpers/CoinConverter.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Numerics;
+using Lykke.Service.Qtum.Api.Core.Services;
+
+namespace Lykke.Service.Qtum.Api.Core.Helpers
+{
+    public class CoinConverter
```

Comment:
I guess it should be static class with static methods. Why do you have to create new instances every time

Reply:
for configurating accuracy?